exercises/linked-list/linked_list.py

```diff
@@ -1,13 +1,8 @@
-# Skeleton file for the Python "linked-list" exercise.
-
-
-class Node(object):
-    def __init__(self, value, next=None, prev=None):
-        self.value = value
-        self.next = next
-        self.prev = prev
+class Node:
+    def __init__(self):
+        pass
 
 
-class LinkedList(object):
+class LinkedList:
```

Comment:
As we support Python 2 & 3 we should subclass from `object` to have new style classes in both versions. That goes for all other cases as well.